docker/Dockerfile.rocm

```diff
@@ -16,6 +16,10 @@ ARG SGL_BRANCH=${SGL_DEFAULT}
 ARG TRITON_REPO="https://github.com/triton-lang/triton.git"
 ARG TRITON_COMMIT="845d75a"
 
+
+ARG ATER_REPO="https://github.com/HaiShaw/ater"
```

Comment:
If this installation method is used, users cannot directly use it by installing `sglang[hip]`, they must use a docker image or install these libraries separately. When do we plan to integrate https://github.com/HaiShaw/ater into `sgl-kernel`, and when should we use the PyPI wheel?

Reply:
BTW if you want to integrate the `ater` library into `sgl-kernel`, the current code style, implementation, comments, and documentation all need optimization. I took a brief look and found that `ater` is still in a very early stage.

Reply:
@zhyncs we will make ATER more official within ROCm, code drop in `sgl-kernel` isn't decided yet, mainly due to its broader use cases. PyPl wheel will be coming soon after some process. This is an incremental one, to whole bunch of commits.